src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,40 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final K[] keys = (K[]) new Object[5];
+    private final V[] values = (V[]) new Object[5];
+    private int count = 0;
```

Comment:
```suggestion
    private int size;
```

it will be initialized by 0 by default